orc8r/cloud/go/mproto/mproto.go

```diff
@@ -14,14 +14,30 @@
 package mproto
 
 import (
+	"crypto/md5"
+	"encoding/base64"
+
 	"magma/orc8r/cloud/go/mproto/protos"
 
 	"github.com/golang/protobuf/proto"
 )
 
-// MarshalManyDeterministic deterministically encodes a slice of protobuf messages,
+// HashManyDeterministic takes a collection of proto messages and returns a deterministic
+// base64-encoded md5 hash of the messages.
+func HashManyDeterministic(protosByID map[string]proto.Message) (string, error) {
+	digestBytes, err := marshalManyDeterministic(protosByID)
+	if err != nil {
+		return "", err
+	}
+	// Convert to a constant-length hash of the encoded value.
+	sum := md5.Sum(digestBytes)
```

Comment:
"digestBytes" isn't a digest yet -- it's just the marshaled bytes. So can call it "marshaled" or "bytes"